maquetado/index.html

```diff
@@ -11,5 +11,15 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1> 
+	
```

Comment:
check intentation and delete extra empty line

Reply:
use two spaces

Reply:
done